packages/ckb-sdk-core/package.json

```diff
@@ -9,14 +9,23 @@
   "typings": "lib/index.d.ts",
   "directories": {
     "lib": "lib",
+    "lib-esm": "lib-esm",
     "umd": "umd",
     "test": "__tests__"
   },
   "files": [
     "lib",
+    "lib-esm",
     "umd",
     "types"
   ],
+  "exports": {
+    ".": {
+      "import": "./lib-esm/index.js",
```

Comment:
`es module` should be named with `.mjs`

Reply:
I don't think the `.mjs` extension is necessary because the filenames already distinguish between ESM and CommonJS, and the entry files for ESM and CommonJS are also specified in the package.json.

The `.js` extension also can work. For example https://www.npmjs.com/package/@ckb-ccc/ccc?activeTab=code

Reply:
Got it